### PR TITLE
support/27745-phan quyen nguoi dung

### PR DESCRIPTION
**Pull request by chienpt98**
[Redmine](https://edu-redmine.sun-asterisk.vn/issues/27745?issue_count=20&issue_position=1&next_issue_id=27744)

### CODE REVIEW

**Comment by chienpt98** [comment]
Screenshot from 2020-08-21 10-53-38

**Review by matTroiCuaHieu — APPROVED** [review]

**Comment by chienpt98** [comment]
ready

File: app/views/users/_user.html.erb [hunk 1]
```diff
@@ -12,12 +12,16 @@
         <span class="badge badge-employee"><%= t ".employee" %></span>
       <% end %>
     </td>
-    <td style = "padding-left: 25px">
-      <% if current_user.admin? && !current_user?(user) %>
-        <%= link_to user, method: :delete, data: { confirm: t(".you_sure") } do %>
-          <i class="fas fa-trash-alt fa-1x "data-toggle="tooltip" data-html="true" title=<%= t(".delete")%>></i>
+    <% if current_user.admin? %>
+      <td style = "padding-left: 25px">
+        <% if !user.admin? %>
```
Comment (hieplt-0914) [inline app/views/users/_user.html.erb]: unless

**Comment by chienpt98** [comment]
ready

**Comment by chienpt98** [comment]
Screencast from 21-08-2020 15_18_59

**Comment by chienpt98** [comment]
ready

**Pull request merged**